rust/protocol/src/identity_key.rs

```diff
@@ -11,6 +16,7 @@ use std::convert::TryFrom;
 
 use prost::Message;
 
+/// Wrapper for [PublicKey].
```

Comment:
Thank you! Fixed!

Reply:
Er, conversely I'd like to remove the reference to IdentityKeyStore, since that's not necessarily the only place such a key is used. The other part makes sense to me, though.